packages/types/src/simple-types.ts

```diff
@@ -19,9 +19,7 @@ export type SolidityValueType =
 // around in protocol messages and include this in transaction data in challenges,
 // we impose some restrictions on the type; they must be serializable both as
 // JSON and as solidity structs.
-export type SolidityABIEncoderV2Type =
-  | SolidityABIEncoderV2Struct
-  | SolidityABIEncoderV2SArray;
+export type SolidityABIEncoderV2Type = SolidityValueType
```

Comment:
Do we need both of these types? Maybe pick one and re-use across code.

Reply:
fixed in #2082